### PR TITLE
Proofread and update `use-bots` documentation

### PR DESCRIPTION
**Pull request by chauhankaranraj**
## Related Issues and Dependencies

None

## This introduces a breaking change

- [ ] Yes
- [x] No

## This Pull Request implements

Fixes link to kebechet and adds some more prose.

### CODE REVIEW

File: docs/source/use-bots.md [hunk 1]
```diff
@@ -1,9 +1,11 @@
...
-If you want to know more about Thoth bots and services have a look [here](https://github.com/AICoE/aicoe-ci/blob/master/docs/thoth-bots-setup.md#instructions-to-setup-bots-and-ci-services).
+If you want to manage your project dependency updates automatically, you can install and set up the [Kebechet bot](https://github.com/thoth-station/kebechet) in your repo. Kebechet regularly checks if your project is using the optimal set of dependencies in terms of  CVE vulnerabilities, newer package releases, and performance changes. If it finds that the dependencies can be updated, it automatically opens a PR on your repo to make the required updates!
+
+An example of the Kebechet bot in action can be seen below.
 
 <div style="text-align:center">
 <img alt="Kebechet Automatic Update" src="https://raw.githubusercontent.com/thoth-station/elyra-aidevsecops-tutorial/master/docs/images/KebechetAutomaticUpdate.png">
 </div>
+
+To learn more about Thoth bots and services, please check out the guide [here](https://github.com/AICoE/aicoe-ci/blob/master/docs/thoth-bots-setup.md#instructions-to-setup-bots-and-ci-services).
```
Comment (isabelizimm) [inline docs/source/use-bots.md:11]: Can you add a "Next Step" with a link to the next notebook?

Comment (oindrillac) [reply docs/source/use-bots.md:11]: +1 🙂

**Review by oindrillac — COMMENTED** [review]
/lgtm

File: docs/source/use-bots.md (hunk 1 above)
Comment (chauhankaranraj) [reply docs/source/use-bots.md]: Sure, added

**Review by oindrillac — APPROVED** [review]
/lgtm !

**Review by isabelizimm — APPROVED** [review]
/lgtm

**Pull request merged**